src/test/java/org/shikshalokam/uiPageObjects/PWBasePage.java

```diff
@@ -42,6 +37,11 @@ public class PWBasePage extends MentorEDBaseTest {
         if (PropertyLoader.PROP_LIST.getProperty("mentor.qa.browser.setHeadless").equals("false")) {
             headless = false;
         }
+        initializeBrowser();
+    }
+
+
+    public static void initializeBrowser() {
```

Comment:
Issues Encountered During Test Suite Execution:
1. Two instances are loading:

-  a. One for Chrome (always), as it’s assigned in the reinitialization method.
-  b. The instance assigned to mentor.qa.browser. 

2. The test suite always runs on Chrome, irrespective of the value of mentor.qa.browser.

Creating a separate initializeBrowser method and calling it within the reinitialization method resolves both issues. This ensures that only one browser instance is loaded, and the test suite runs on the specified mentor.qa.browser value.